### PR TITLE
Update Embed Optimizer to be standalone plugin (along with merge from trunk)

### PR DESCRIPTION
**Pull request by westonruter**
Beyond just merging `trunk` into `feature/lazy-embeds` (b76441791ac63cae2dab6195dd4e2f769ae58369), this PR then applies the following changes (b76441791ac63cae2dab6195dd4e2f769ae58369...61ad4959296112f021bbb158102e15ec5ae52e4c) to complete the change from a module to a plugin.

Compare diff with `trunk`: 7ba61f4...61ad4959296112f021bbb158102e15ec5ae52e4c ([changes](https://github.com/WordPress/performance/compare/7ba61f447645bf2d794d329d9e1b1b8abf76aaf2...61ad4959296112f021bbb158102e15ec5ae52e4c)).

### CODE REVIEW

File: plugins/embed-optimizer/load.php [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * Plugin Name: Embed Optimizer
+ * Plugin URI: https://github.com/WordPress/performance/issues/888
```
Comment (mukeshpanchal27) [inline plugins/embed-optimizer/load.php]:
```suggestion
 * Plugin URI: https://github.com/WordPress/performance/tree/trunk/plugins/embed-optimizer
```

File: plugins/embed-optimizer/readme.txt [hunk 2]
```diff
@@ -0,0 +1,52 @@
+=== Embed Optimizer ===
+
+Contributors:      wordpressdotorg
+Requires at least: 6.3
+Tested up to:      6.5
+Requires PHP:      7.0
+Stable tag:        0.1.0
+License:           GPLv2 or later
+License URI:       https://www.gnu.org/licenses/gpl-2.0.html
+Tags:              performance, images
```
Comment (mukeshpanchal27) [inline plugins/embed-optimizer/readme.txt]: Tags should be changed for this plugin `embeds`?

File: plugins/embed-optimizer/hooks.php [hunk 3]
```diff
@@ -136,6 +136,8 @@ function embed_optimizer_lazy_load_scripts() {
  *
  * Generates the message when `WP_DEBUG` is true.
  *
+ * @since n.e.x.t
```
Comment (mukeshpanchal27) [inline plugins/embed-optimizer/hooks.php]: Update with actual version number.

Comment (westonruter) [reply plugins/embed-optimizer/hooks.php]: Done in 8388bb9

File: plugins/embed-optimizer/readme.txt (hunk 2 above)
Comment (westonruter) [reply plugins/embed-optimizer/readme.txt]: Done in 6b8d72a

File: plugins/embed-optimizer/load.php (hunk 1 above)
Comment (westonruter) [reply plugins/embed-optimizer/load.php]: Done in f5d83a4

**Review by mukeshpanchal27 — APPROVED** [review]

**Pull request merged**